src/form/DecompositionTable/DecompositionTable.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { TableHeader, TableBody, TableRow, TableCell, Switch, ContextMenuItem } from '@amsterdam/asc-ui';
+import { Ellipsis } from '@amsterdam/asc-assets';
+import { StyledDecompositionTable, StyledContextMenu } from './DecompositionTableStyles'
+
+
+export type Props = {
+	rows: any[];
+};
+
+const DecompositionTable: React.FC<Props> = ({ rows }: Props) => {
+
+	return (
+		<StyledDecompositionTable>
+			<TableHeader>
+				<TableRow>
+					<TableCell as="th">Code</TableCell>
+					<TableCell as="th">Deel</TableCell>
+					<TableCell as="th">Materiaal</TableCell>
+					<TableCell as="th">Hoeveelheid</TableCell>
+					<TableCell as="th">Eenheid</TableCell>
+					<TableCell as="th">Locatie</TableCell>
+					<TableCell as="th">Jaar</TableCell>
+					<TableCell as="th">Relevant</TableCell>
+					<TableCell as="th">Acties</TableCell>
+				</TableRow>
+			</TableHeader>
+			<TableBody>
+				{rows.map((row) => (
```

Comment:
I believe best practice dictates that a map like this should render a standalone component again. Especially considering that the complexity in a TableRow could increase somewhat over time, I'd create a dedicated component for rendering a table row.